tests/common/snappi/snappi_fixtures.py

```diff
@@ -517,3 +521,373 @@ def cvg_api(snappi_api_serv_ip,
     yield api
     if getattr(api, 'assistant', None) is not None:
         api.assistant.Session.remove()
+
+
+def snappi_dut_base_config(duthost_list,
+                           snappi_ports,
+                           snappi_api):
+    """
+    Generate snappi API config and port config information for the testbed
+    Args:
+        duthost_list (pytest fixture): list of DUTs
+        snappi_ports: list of snappi ports
+        snappi_api(pytest fixture): Snappi API fixture
+    Returns:
+        - config (obj): Snappi API config of the testbed
+        - port_config_list (list): list of port configuration information
+    """
+
+    """ Generate L1 config """
+
+    config = snappi_api.config()
+    tgen_ports = [port['location'] for port in snappi_ports]
```

Comment:
Rename this to "snappi_ports"?

Reply:
this will addressed in a separate PR , an issue has been raised to track this - > https://github.com/sonic-net/sonic-mgmt/issues/8210